recipe/meta.yaml

```diff
@@ -627,7 +629,11 @@ outputs:
       run:
         - python # dummy dependency to create python-version dependent package
         - {{ pin_subpackage('libopenvino-dev', exact=True) }}
-        - {{ pin_subpackage('libopenvino-python', exact=True) }}
+        # Do not use the exact pin here since we are generating multiple different 
+        # python packages in 1 go. Instead let the common python constraints
+        # choose the same variant of the package
+        # - {{ pin_subpackage('libopenvino-python', exact=True) }}
+        - {{ pin_subpackage('libopenvino-python', max_pin='x.x.x') }}
```

Comment:
should we remove build string for `openvino` meta package then? Currently, it contains `py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}`

Reply:
i think it is fine to have 4 of these, but they could be collapsed if you prefer.